Package.swift

```diff
@@ -35,7 +34,7 @@ let package = Package(
         .package(url: "https://github.com/StanfordSpezi/SpeziFoundation", from: "2.0.0"),
         .package(url: "https://github.com/StanfordSpezi/SpeziStorage", from: "1.0.2"),
         .package(url: "https://github.com/StanfordSpezi/SpeziOnboarding", from: "1.1.1"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziChat", .upToNextMinor(from: "0.2.1")),
+        .package(url: "https://github.com/StanfordSpezi/SpeziChat", .upToNextMinor(from: "0.2.2")),
         .package(url: "https://github.com/StanfordSpezi/SpeziViews", from: "1.3.1")
```

Comment:
We should try to unify this across all packages, some of them use `.git` at the end, some of them don't. I would suggesting adding `.git` for all of them for now.